tests/D2L.CodeStyle.Analyzers.Test/Threading/UnsafeAwaitTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using D2L.CodeStyle.Analyzers.Test.Verifiers;
+using D2L.CodeStyle.Analyzers.UnsafeStatics;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using NUnit.Framework;
+
+namespace D2L.CodeStyle.Analyzers.Threading {
+
+	internal sealed class UnsafeAwaitTests : DiagnosticVerifier {
+
+		protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer() {
+			return new UseConfigureAwaitFalseAnalyzer();
+		}
+
+		[Test]
+		public void VerifyAnalyzer_HasUnsafeAwait_DiagnosticsShowUp() {
+
+			const string code = @"using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+public class Test {
+	Task<int> DoSomethingAsync() {   
+		return Task.FromResult(5);
+	}
+
+	async Task<string> DoSomethignElseAsync() {
+		int i = await DoSomethingAsync();
+		return $""hello {i}th person"";
+	}
+}";
+
+			AssertSingleDiagnostic(code, 11, 10, "bad", "it");

```

Comment:
Ah okay sorry I'll jump off this until you're good. I thought maybe you'd forgotten to attach a reviewer. 

Ping/attach me when you're good.